tasks/modprobe.yml

```diff
@@ -14,6 +14,12 @@
     os_unused_filesystems: "{{ os_unused_filesystems | difference('vfat') }}"
   when: efi_installed.stat.isdir is defined and efi_installed.stat.isdir
 
+# some systems seem to require /boot/efi mounted while not exposing /sys/firmware/efi
+- name: remove vfat from fs-list if /boot/efi is mounted
```

Comment:
Why not aff then condition to the task before?

Reply:
i wanted to separate that, because I don't really like this solution and I thought an or condition might complicate things